examples/blueprint-construct/index.ts

```diff
@@ -65,6 +65,7 @@ export default class BlueprintConstruct {
             }),
             new blueprints.addons.XrayAdotAddOn(),
             // new blueprints.addons.CloudWatchAdotAddOn(),
+            // new blueprints.addons.ContainerInsightsAddOn(),
```

Comment:
Hmm, we have a few incompatible add-ons. We need to create a new e2e test for let's say CW observability with Cluster Autoscaler (currently incompatible with Karpenter). This can be deferred though.